### DISCUSSION TITLE
Living Light with Netatmo & HA

### DISCUSSION THREAD

**Discussion by esposimo** [indomus/forum#1544] [HUB: Home Assistant]
Salve a tutti,

come da titolo, qualcuno qui ha provato ad integrare la serie bTicino living light con Home Assistant ?

Dal sito bTicino vedo che anche la living light ha i suoi dispositivi smart (compresi i moduli wireless e il DIn per il carico nel quadro elettrico)
Sono ugualmente compatibili con home assistant come la living now (che se ben ricordo va disassociata dall'app per essere vista da HA)

**Comment by asjbm** [comment]
Ciao, il sistema è lo stesso, cambia solo la cover (già applicata per la serie Light e customizzabile per la Now).
Solo il modulo DIN per il controllo carico fa un po' storia a sè, nel senso che se usi il bridge di BTicino con l'integrazione "Controller HomeKit" in HA il modulo non viene "riportato" in HA (almeno io anche abilitando tutto nell'integrazione non sono riuscito a vederlo come sensore), mentre dovrebbe essere possibile integrarlo usando Conbee (o Rasbee).

Per questo motivo sto passando ad una integrazione "ibrida", usando Conbee per il modulo DIN e il controller HomeKit per tutto il resto.

Ciao

**Reply by esposimo** [reply]
avrei un paio di domande
1) il carico, lo vedi totale per la casa, o lo vedi singolo per ogni presa?
2) Ho letto che per avere tutto su HA è necessario eliminare l'associazione del gateway con l'app su smartphone. Hai provato eventualmente con questa operazione ?

Grazie :)

**Reply by asjbm** [reply]
Rispondendo per punti:

1. il DIN ha una singola sonda che monitora la fase dell'interruttore generale, quindi restituisce la misurazione del carico totale per la casa; per avere il dettaglio delle prese dovresti usare un modulo presa connesso per ogni presa che vuoi monitorare; questo però si porta via lo spazio di un frutto, quindi ha senso metterlo per monitorare i principali elettrodomestici non domotici (tranne il piano a induzione, che richiede un carico ben più alto di quello sopportato dal modulo connesso), per poterne poi anche controllare l'eventuale spegnimento in caso di superamento del limite contrattuale, o se vuoi automatizzare l'accensione si una stufetta o uso simile, non è tanto sensato l'utilizzo su tutte le prese

2. in realtà no, usando l'integrazione "Controller Homekit" si riesce a far vedere il gateway BTicino a Home Assitant, e di conseguenza a esporre su quest'ultimo i vari interruttori, comandi tapparelle (con percentuale apertura) e prese connesse, i quali restano quindi nella rete zigbee del gateway e quindi sono visibili e gestibili contemporaneamente anche da Home + Connect
 
Il limite di questa soluzione, come detto, è che non vengono esposti su HA le entità relative ai sensori di assorbimento, quindi i moduli presa connessa vengono visti come "prese comandate" ma senza info sull'assorbimento, e il DIN non viene proprio rilevato come entità (a meno che un domani non venga integrata l'esposizione come entità di tali informazioni).

Detto questo (a parte l'abbondantemente discussa questione di avere parte dei propri sistemi casalinghi in cloud) il punto fondamentale è qual'è l'obiettivo che vuoi raggiungere, quanto ci vuoi spendere, quanto tempo vuoi/puoi investirci, e quanto saranno eterogenei i device domotici che intendi metterti in casa (marche diverse, standard diversi, ecc).

Se l'impianto di BTicino ti è sufficiente e ti interessa solo buttare un occhio ogni tanto per curiosità ai consumi, Home + Control ti da visibilità sia di quello generale che degli eventuali singoli moduli connessi che hai collegato (es, fatto 100 il consumo totale, lui ti presenta i consumi esatti dei moduli presa connessi e il resto va in una voce unica come "resto della casa"), e lo implementi con il minimo sforzo.

Se hai un problema di distacco della corrente e vuoi limitare queste situazioni, può essere sensata la soluzione ibrida (quella che sto cercando di implementare io), aggiungendo i 40/50€ per una Conbee2 (o Raspbee) e collegando il DIN controllo carichi i moduli presa connessa su HA usando l'integrazione ZHA per andare a spegnere le prese che stanno effettivamente generando consumo, e lasciare tutto il resto al gateway (luci e tapparelle) per comodità, e questo già richiede decisamente un maggior impegno, per scrivere o riadattare le configurazioni e gli script che si trovano in rete.

L'estremo opposto infine è quello di non avere nulla su gateway (soluzione sensata se non l'hai ancora comprato), a quel punto configuri tutto tramite Conbee, hai un monitoraggio e uno storico preciso dei singoli consumi, ogni modulo connesso è gestito direttamente, ma ovviamente si porta dietro una maggior complessità della configurazione e più tempo da dedicargli.

**Reply by esposimo** [reply]
Ti ringrazio innanzitutto per la spiegazione esaustiva. Il mio obiettivo è controllare (consumi, accensione/spegnimento) di tutto, prese, luci, tapparelle, da HA e non da app ufficiale (non per forza, per intenderci).
Mi rendo conto che per ogni presa porto via un frutto, ma dato che sarò in ristrutturazione completa, sto cercando di predispormi prima per evitare lavori futuri. Magari non metto tutti i sensori sin da subito se il costo è elevato, ma essendo l'impianto predisposto li metto poco alla volta. Mi hanno sconsigliato gli shelly (sempre uno per presa, o magari 1 per quattro prese come il 4 Pro) pur sfruttando quelli che hanno la porta eth (quindi evitando molti dispositivi in wifi) perchè ci sarebbe da passare una marea di cavi in più.
Credo che la soluzione più idonea sia la 3, ovvero tutto su Conbee. La parte di configurazione non mi preoccupa, lavorando nel ramo IT dovrebbe venirmi + "facile".
Per quanto riguarda i dati in cloud, non dovrebbe essere un problema. Creo due vlan sul router che ho, una per i dispositivi IoT e una per i dispositivi personali (pc, smartphone, etc) e blocco il traffico in uscita per la vLan IoT. HA vedrà entrambi le reti, così può vedere tutti i dispositivi IoT ma essere visto anche dal resto della casa che può gestire i vari dispositivi attraverso di esso. Non credo che senza traffico in uscita i dispositivi non funzionino, non con i metodi ufficiali insomma (es. app di bTicino)

**Comment by piero79** [comment]
ciao a tutti, io ho appena ristrutturato casa (per il momento non ci vivo ancora, ma mancano poche settimane) e ho integrato la serie Bticino living now su HA tramite HomeKit controller.
Ho lasciato installata sul telefono l'app "Control" della Bticino e ho solo messo il codice del gateway su HomeKit controller.
Per ora vedo tutto tranne 2 tapparelle elettriche su 4....sapete quale possa essere il problema?

Grazie
